CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Each line is a file pattern followed by one or more owners.
+# More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+
+*         @hashicorp/vault-devex
```

Comment:
I'm not seeing this in the docs. Does this require someone from vault-devex to approve a PR? if not, that may be a good next thing to add.

Reply:
Good idea. This file does not automatically do that, but I just added it to our repo settings.